### PR TITLE
modules 55, 56, 58, 61, 62, 65, 70, 73, 74, 76, 79, 80, 81, 82, 85 and 86 fully shiftable

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

**Comment by red031000** [comment]
ALL MODULES SHIFTABLE

File: arm9/modules/79/asm/module_79_thumb.s [hunk 1]
```diff
@@ -897,11 +897,58 @@ MOD79_02216834: ; 0x02216834
 
 	.global MOD79_0221684C
 MOD79_0221684C: ; 0x0221684C
-	.byte 0x25, 0x73, 0x3F, 0x70, 0x69, 0x64, 0x3D, 0x25, 0x64, 0x00, 0x00, 0x00, 0x62, 0x75, 0x66, 0x66
+	.byte 0x25, 0x73, 0x3F, 0x70, 0x69, 0x64, 0x3D, 0x25, 0x64, 0x00, 0x00, 0x00
```
Comment (PikalaxALT) [inline arm9/modules/79/asm/module_79_thumb.s:900]: `.asciz "%s?pid=%d"` (word-aligned)

File: arm9/modules/79/asm/module_79_thumb.s [hunk 2]
```diff
@@ -897,11 +897,58 @@ MOD79_02216834: ; 0x02216834
...
+
+	.global MOD79_02216858
+MOD79_02216858: ; 0x02216858
+	.byte 0x62, 0x75, 0x66, 0x66
```
Comment (PikalaxALT) [inline arm9/modules/79/asm/module_79_thumb.s:904]: `.asciz "bufferIn != NULL"` (word-aligned)

**Comment by red031000** [comment]
done

**Pull request merged**